tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -61,8 +62,10 @@
 #define BTP_SERVICE_ID_VCP	20
 #define BTP_SERVICE_ID_CAS	21
 #define BTP_SERVICE_ID_MCP	22
+#define BTP_SERVICE_ID_GMCS	23
+#define BTP_SERVICE_ID_MCS	24
```

Comment:
So in the test spec/test suite these 2 services with (near?) identical behavior will be split?

Reply:
Exactly, tests are split to GMCS and MCS with identical behavior.